push/src/main/java/org/aerogear/mobile/push/PushService.java

```diff
@@ -34,75 +35,102 @@
 import org.aerogear.mobile.core.reactive.Requester;
 import org.aerogear.mobile.core.reactive.Responder;
 
-
 /**
  * The entry point for communication with Unified Push Server
  */
-public class PushService implements ServiceModule {
+public class PushService {
```

Comment:
Why are we removing ServiceModule?

Reply:
@secondsun because it forced me to implement some methods used by MobileCore I wanna kill, but I don't wanna touch on MobileCore/ServiceModule for now.

Reply:
@wtrocki @danielpassos I was thinking it could still be a marker interface for our tooling, but we don't have any tooling, nvm :)

Reply:
We can re introduce the interface later if we want